c.pod_design.md

```diff
@@ -83,6 +83,8 @@ kubectl label po nginx1 nginx2 nginx3 app-
 kubectl label po nginx{1..3} app-
 # or
 kubectl label po -lapp app-
+#or
+kubectl label po --all app-
```

Comment:
This will remove the label from *all* Pods, right? This may be correct here but may be wrong in another context (e.g. if we add more Pods)

Reply:
Yes, it will remove label _app from all the pods in that namespace.